go/runtime/host/tdx/qemu.go

```diff
@@ -266,6 +272,45 @@ func (p *qemuProvisioner) getSandboxConfig(cfg host.Config, _ sandbox.Connector,
 	return pcfg, nil
 }
 
+// createNetworkingConfig generates QEMU networking configuration for a component.
+func (p *qemuProvisioner) createNetworkingConfig(cfg host.Config) ([]string, error) {
+	compCfg, _ := config.GlobalConfig.Runtime.GetComponent(cfg.ID, cfg.Component.ID())
+	netdevOpts := []string{"user", "id=net0"}
+
+	// Inbound forwarding.
+	for _, netCfg := range compCfg.Networking.Incoming {
```

Comment:
Should we validate that same ip,protocol,source port combination doesn't repeat among the rules? 

Otherwise we will probably get (a not very nice) error on quemu start.

Reply:
Added some config validation, please take a look.